app/services/participants/history_builder.rb

```diff
@@ -124,7 +124,10 @@ def record_participant_declaration_events(declarations)
 
       declaration.declaration_states.each do |declaration_state|
         type = "#{declaration.declaration_type.capitalize}Declaration"
-        actor = declaration.cpd_lead_provider.name
+
+        actor = declaration.voided_by_user if declaration.voided?
```

Comment:
should we also include awaiting clawback/clawed back?

Reply:
Ah yes, so looking at the services we set the user when it moves to `awaiting_clawback` as well as `voided`, but I don't think it applies to `clawedback` as that happens via a job? I'll update it for `awaiting_clawback, let me know if that makes sense to leave `clawedback` as-is